t/40-job_settings.t

```diff
@@ -47,7 +47,7 @@ my $settings = {
     HDDSIZEGB                => 50,
     INSTALLONLY              => 1,
     PATTERNS                 => 'base,minimal',
-    PUBLISH_PFLASH_VARS =>
+    PUBLISH_PFLASH_VARS      =>
       'SLES-%VERSION%-%ARCH%-%BUILD%@%MACHINE%-minimal_with_sdk%BUILD_SDK%_installed-uefi-vars.qcow2',
     SEPARATE_HOME      => 0,
```

Comment:
And on next release it will align this arrow too?

Reply:
one at a time, don't rush it :D

I guess this shows all the critics that Perl is far from a dead language, right? ;)